esp/scm/ws_packageprocess.ecm

```diff
@@ -321,6 +323,7 @@ ESPrequest AddPartToPackageMapRequest
     [min_ver("1.05")] bool StopIfFilesCopied(false); //Command only completes if no files need copying.  User can run again after DFU Publisher Workunit completes.
     [min_ver("1.06")] string DfuPublisherWuid; //Wuid can be preallocated and then passed in here to use.  Will be created if empty
     [min_ver("1.06")] string RemoteStorage;
+    [min_ver("1.06")] string keyCompression;
```

Comment:
The ESP service has been updated to 1.07. Should the min_ver for KeyCompression be 1.07.